src/components/pager/pager.stories.tsx

```diff
@@ -1,13 +1,40 @@
 import React from "react";
-import { ComponentStory } from "@storybook/react";
+import { Meta, StoryObj } from "@storybook/react";
 
 import Pager, { PagerProps } from ".";
 import useMediaQuery from "../../hooks/useMediaQuery";
 
-export const Default: ComponentStory<typeof Pager> = (args: PagerProps) => (
-  <Pager {...args} />
-);
+const meta: Meta<typeof Pager> = {
+  title: "Pager",
+  component: Pager,
+};
+
+export default meta;
+type Story = StoryObj<typeof Pager>;
+
+export const DefaultStory: Story = {
+  render: (args: PagerProps) => {
+    return <Pager {...args} />;
+  },
+  args: {
+    totalRecords: "100",
+    showPageSizeSelection: false,
+    currentPage: "1",
+    pageSizeSelectionOptions: [
+      { id: "1", name: 1 },
+      { id: "10", name: 10 },
+      { id: "25", name: 25 },
+      { id: "50", name: 50 },
+      { id: "100", name: 100 },
+    ],
+  },
+  name: "Default",
+};
 
+export const Default: Story = (args: PagerProps) => {
+  return <Pager {...args} />;
+};
```

Comment:
DefaultStory was originally being used when I was experimenting how to generate the other stories instead of the `.bind()` method but was never tidied up. Removing now.